app/decorators/corporate_decorator.rb

```diff
@@ -21,6 +21,16 @@ def extra_content?
   private
 
   def extra_content_partial_name
+    return 'tool' if slug_is_corporate_tool_directory?
```

Comment:
is it not possible to have another decorator which then implements the method `extra_content_partial_name ` instead of adding if statements?